BuildingThemes/BuildingThemesMod.cs

```diff
@@ -1,191 +1,28 @@
 ﻿using ICities;
-using ColossalFramework;
-using UnityEngine;
-using System;
 using BuildingThemes.GUI;
-using ColossalFramework.UI;
-using Object = UnityEngine.Object;
+using UnityEngine;
 
 namespace BuildingThemes
 {
-    public class BuildingThemesMod : LoadingExtensionBase, IUserMod
+    public class BuildingThemesMod : IUserMod
     {
-        public string Name
-        {
-            get { return "Building Themes"; }
-        }
-        public string Description
-        {
-            get { return "Create building themes and apply them to cities and districts."; }
-        }
+        // we'll use this variable to pass the building position to GetRandomBuildingInfo method. It's here to make possible 81 Tiles compatibility
+        public static Vector3 position;
```

Comment:
I moved position here because 81 Tiles should find a class with a unique name ```BuildingThemesMod``` to set its static field